genetics.pde

```diff
@@ -14,6 +14,10 @@ LinkedList<int[]> water;
 int waterInitAmount = 300;
 color waterColor = color(0, 39, 195);
 
+//This is the size of each initial critter. According to current implementation, offspring will be half the size.
+//critterSize is functional, and larger critters will eat more food at a time.
+static final int critterSize = 40;
```

Comment:
Please put final variables in all caps, like CRITTER_SIZE
I also think this constant would be better located in critter, as Critter.CRITTER_SIZE

Reply:
I'm curious why you think CRITTER_SIZE should be located in critter? I feel like a constant would be best placed in genetics, and within critter would be stored an instance variable of whatever size an individual critter decides to have. (Assuming a framework where size changes, which seems to be where the club is moving)

Reply:
It's a good point. I think you're right, it's better off in `genetics` to represent the "actual" critter size, and if the class insists on having some other notion of size, it won't effect its eating/drinking/mating radius